lib/open_food_network/enterprise_injection_data.rb

```diff
@@ -1,7 +1,7 @@
 module OpenFoodNetwork
   class EnterpriseInjectionData
     def active_distributors
-      @active_distributors ||= Enterprise.distributors_with_active_order_cycles
+      @active_distributors ||= Enterprise.distributors_with_active_order_cycles.andand.ready_for_checkout
```

Comment:
Since `distributors_with_active_order_cycles` is a scope, I don't think you need `andand`.